### DISCUSSION TITLE
How can I get the driver to work?

### DISCUSSION THREAD

**Discussion by alemunter** [berarma/new-lg4ff#61] [Q&A] [answered]
Hi! I installed this driver following all the instructions with DKMS. But when I run `sudo dmesg`, I get this

`[  519.067699] usb 1-4: new full-speed USB device number 4 using xhci_hcd`
`[  519.378602] usb 1-4: New USB device found, idVendor=046d, idProduct=c267, bcdDevice=38.00`
`[  519.378605] usb 1-4: New USB device strings: Mfr=1, Product=2, SerialNumber=3`
`[  519.378606] usb 1-4: Product: G923 Racing Wheel for PlayStation 4 and PC`
`[  519.378607] usb 1-4: Manufacturer: Logitech`
`[  519.378608] usb 1-4: SerialNumber: COWYTEZUNUZYNARYNUBOVEKUVU`
`[  519.410466] hid-generic 0003:046D:C267.0006: ignoring exceeding usage max`
`[  519.410615] input: Logitech G923 Racing Wheel for PlayStation 4 and PC as /devices/pci0000:00/0000:00:01.3/0000:02:00.0/usb1/1-4/1-4:1.0/0003:046D:C267.0006/input/input25`
`[  519.410694] hid-generic 0003:046D:C267.0006: input,hiddev99,hidraw5: USB HID v1.10 Gamepad [Logitech G923 Racing Wheel for PlayStation 4 and PC] on usb-0000:02:00.0-4/input0`
`[  519.418393] hid-generic 0003:046D:C267.0007: hiddev100,hidraw6: USB HID v1.11 Device [Logitech G923 Racing Wheel for PlayStation 4 and PC] on usb-0000:02:00.0-4/input1`
`[  519.426385] hid-generic 0003:046D:C267.0008: hiddev101,hidraw7: USB HID v1.11 Device [Logitech G923 Racing Wheel for PlayStation 4 and PC] on usb-0000:02:00.0-4/input2`

what am I doing wrong? in oversteer nothing show up.

**Comment by berarma** [comment, marked as answer]
You need #50. Use that branch, and if possible, please test it and provide feedback.
